code/game/turfs/simulated/walls.dm

```diff
@@ -296,12 +285,22 @@
 	if(LAZYLEN(dent_decals) >= MAX_DENT_DECALS)
 		return
 
-	var/mutable_appearance/decal = pick(dent_decal_list[denttype])
+	var/mutable_appearance/decal = mutable_appearance('icons/effects/effects.dmi', "", BULLET_HOLE_LAYER)
```

Comment:
why are you now constructing it every time instead of picking from the static list?

Reply:
I think that reusing the same constructed mutable_appearance was messing up with things. Also the static list thing just looks awful

Reply:
yeah but how does this affect the performance difference?